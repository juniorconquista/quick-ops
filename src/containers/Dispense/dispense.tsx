import React, { memo } from 'react';
import { Box, H1 } from 'quarks-ui';

import Form from './Form';
import { DispenseProps } from './dispense.types';

const Dispense: React.FC<DispenseProps> = (props) => (
    <Box
        styling="column"
        minHeight="439px"
        width="100%"
        borderRadius="6px"
        bg="white.contrast"
        mt="24px"
        justifyContent="flex-start"
        alignItems="flex-start"
    >
        <H1
            styling="base"
            color="text.contrast"
            fontWeight="bold"
            fontSize={20}
            p={{ mobile: '0px 15px', tablet: '0px 15px', desktop: '0px 48px' }}
            pt={{ mobile: '15px', tablet: '15px', desktop: '24px' }}
        >
            Nova despesa
        </H1>
        <Form handleDispense={props.handleDispense} />
    </Box>
);

export default memo(Dispense);
