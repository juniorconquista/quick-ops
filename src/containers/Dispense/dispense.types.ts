export interface DispenseProps {
    handleDispense: () => void;
}
