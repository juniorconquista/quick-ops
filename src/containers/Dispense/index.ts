import Dispense from './dispense';

export default Dispense;
