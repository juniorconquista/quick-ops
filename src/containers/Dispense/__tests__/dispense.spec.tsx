import React from 'react';
import { cleanup, render } from '@testing-library/react';

import ThemeProvider from '../../../utils/ThemeProviderTest';
import Dispense from '../dispense';
import 'jest-styled-components';

const props = {
    handleDispense: jest.fn(),
};

describe('Dispense container', () => {
    afterEach(() => {
        jest.clearAllMocks();
        cleanup();
    });

    it('tests basic snapshot', () => {
        const container = render(
            <ThemeProvider>
                <Dispense {...props} />
            </ThemeProvider>,
        );
        expect(container.asFragment()).toMatchSnapshot();
    });
});
