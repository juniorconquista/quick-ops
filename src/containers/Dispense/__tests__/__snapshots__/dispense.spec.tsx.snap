// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dispense container tests basic snapshot 1`] = `
<DocumentFragment>
  .c1 {
  color: #f8f9fa;
  font-size: 2rem;
  color: #FFFFFF;
  font-weight: bold;
  font-size: 20px;
}

.c6 {
  color: #FFFFFF;
  font-weight: bold;
  margin: 12px 0px;
}

.c8 {
  color: #f8f9fa;
  font-size: 1rem;
  color: #848a91;
  text-align: left;
  margin-top: 10px;
}

.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #313438;
  margin-top: 24px;
  position: relative;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  min-height: 439px;
  width: 100%;
  border-radius: 6px;
}

.c0:focus {
  outline: none;
}

.c2 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-items: stretch;
  -webkit-box-align: stretch;
  -ms-flex-align: stretch;
  align-items: stretch;
  width: 100%;
  height: 100%;
}

.c2:focus {
  outline: none;
}

.c3 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #25262a;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  width: 100%;
}

.c3:focus {
  outline: none;
}

.c4 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  width: 100%;
}

.c4:focus {
  outline: none;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-align-items: flex-start;
  -webkit-box-align: flex-start;
  -ms-flex-align: flex-start;
  align-items: flex-start;
  width: 100%;
}

.c5:focus {
  outline: none;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-flex: 1;
  -ms-flex: 1;
  flex: 1;
  -webkit-box-pack: start;
  -webkit-justify-content: flex-start;
  -ms-flex-pack: start;
  justify-content: flex-start;
  width: 100%;
}

.c9:focus {
  outline: none;
}

.c10 {
  display: grid;
  -webkit-flex-direction: column;
  -ms-flex-direction: column;
  flex-direction: column;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: start;
  -webkit-box-align: start;
  -ms-flex-align: start;
  align-items: start;
  grid-template-columns: repeat(2,1fr);
  grid-gap: 20px;
  width: 100%;
}

.c10:focus {
  outline: none;
}

.c14 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-direction: row;
  -ms-flex-direction: row;
  flex-direction: row;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #25262a;
  margin-top: 20px;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  height: 77px;
  width: 100%;
}

.c14:focus {
  outline: none;
}

.c15 {
  height: 36px;
  border-radius: 6px;
  color: #6b7480 !important;
  border: 1px solid;
  border-color: #6b7480;
  padding: 0px 20px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #25262a;
  cursor: pointer;
  -webkit-transition: boxShadow .25s ease;
  transition: boxShadow .25s ease;
  font-family: Nunito Sans,sans-serif;
  color: #FFFFFF;
}

.c15 svg {
  height: 15px;
  width: 15px;
  margin-right: 5px;
  fill: #6b7480;
}

.c15:hover,
.c15:focus {
  box-shadow: 0 0 2px 3px #6b748090;
}

.c16 {
  height: 36px;
  border-radius: 6px;
  color: #FFFFFF;
  border: 1px solid;
  border-color: #51c1c3;
  padding: 0px 20px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #51c1c3;
  cursor: pointer;
  -webkit-transition: boxShadow .25s ease;
  transition: boxShadow .25s ease;
  font-family: Nunito Sans,sans-serif;
  color: #FFFFFF;
  margin: 0px 24px;
}

.c16 svg {
  height: 15px;
  width: 15px;
  margin-right: 5px;
  fill: #FFFFFF;
}

.c16:hover,
.c16:focus {
  box-shadow: 0 0 2px 3px #51c1c390;
}

.c12 {
  width: 100%;
}

.c12 label {
  margin: 12px 0px;
  color: #FFFFFF;
  font-weight: bold;
  display: block;
}

.c12 input {
  height: 36px;
  border-radius: 6px;
  color: #6b7480 !important;
  border: 1px solid;
  border-color: #dfe2e6;
  padding: 12px 6px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #25262a;
  cursor: pointer;
  width: 100%;
  -webkit-transition: boxShadow .25s ease;
  transition: boxShadow .25s ease;
  font-family: Nunito Sans,sans-serif;
}

.c12 input:hover,
.c12 input:focus {
  box-shadow: 0 0 2px 3px #6b748090;
  outline: none;
}

.c11 {
  width: 100%;
}

.c11 label {
  margin: 12px 0px;
  color: #FFFFFF;
  font-weight: bold;
  display: block;
}

.c11 .select__input {
  color: #6b7480;
  font-family: Nunito Sans,sans-serif;
  font-weight: normal;
}

.c11 .select__input > div {
  font-family: Nunito Sans,sans-serif !important;
  font-weight: normal !important;
}

.c11 .select__control {
  border: 1px solid;
  border-radius: 6px;
  background-color: #25262a;
  border-color: #dfe2e6;
  font-family: Nunito Sans,sans-serif;
  height: 36px;
  -webkit-transition: inherit;
  transition: inherit;
}

.c11 .select__control:hover {
  box-shadow: 0 0 2px 3px #6b748090;
  outline: none;
  border: 1px solid;
  border-color: #dfe2e6;
}

.c11 .select__placeholder {
  color: #6b7480;
  font-family: Nunito Sans,sans-serif;
}

.c11 .select__single-value {
  color: #6b7480;
  font-family: Nunito Sans,sans-serif;
}

.c11 .select__control--is-focused {
  box-shadow: 0 0 2px 3px #6b748090;
  border-color: #dfe2e6;
}

.c11 .select__menu {
  background-color: #25262a;
  border: 1px solid;
  border-color: #dfe2e6;
  margin-top: 4px;
  overflow: hidden;
}

.c11 .select__menu-list {
  color: #6b7480;
  font-family: Nunito Sans,sans-serif;
  padding-top: 0rem;
  padding-bottom: 0rem;
}

.c11 .select__option {
  -webkit-transition: all 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  transition: all 200ms cubic-bezier(0.4,0,0.2,1) 0ms;
  line-height: normal;
}

.c11 .select__option:hover {
  background-color: #313438;
}

.c11 .select__option--is-focused {
  background-color: #313438;
}

.c11 .select__option--is-selected {
  background-color: #51c1c3;
}

.c13 {
  width: 100%;
}

.c13 .react-datepicker-wrapper {
  width: 100%;
}

.c13 .react-datepicker-wrapper input {
  height: 36px;
  border-radius: 6px;
  color: #6b7480 !important;
  border: 1px solid;
  border-color: #dfe2e6;
  padding: 12px 6px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  background-color: #25262a;
  cursor: pointer;
  width: 100%;
  -webkit-transition: boxShadow .25s ease;
  transition: boxShadow .25s ease;
  font-family: Nunito Sans,sans-serif;
}

.c13 .react-datepicker-wrapper input:hover,
.c13 .react-datepicker-wrapper input:focus {
  box-shadow: 0 0 2px 3px #6b748090;
  outline: none;
}

.c13 .react-datepicker__current-month,
.c13 .react-datepicker__day-name,
.c13 .react-datepicker__day {
  font-family: Nunito Sans,sans-serif;
  color: #f8f9fa;
}

.c13 .react-datepicker__triangle {
  display: none;
}

.c13 .react-datepicker__header {
  background-color: #25262a;
}

.c13 .react-datepicker__month {
  background-color: #313438;
  margin: 0px;
  padding: 0.4rem;
  overflow: hidden;
}

.c13 .react-datepicker__day:hover {
  background-color: #25262a;
  color: #f8f9fa;
}

.c13 .react-datepicker__day--selected {
  background-color: #51c1c3;
  color: #FFFF;
}

.c13 .react-datepicker__day--selected:hover {
  background-color: #51c1c3;
  color: #FFFF;
}

.c7 {
  height: 208px;
  width: 100%;
  border-radius: 6px;
  color: #6b7480;
  border: 1px dashed;
  border-color: #dfe2e6;
  padding: 0px 20px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: flex-end;
  -webkit-box-align: flex-end;
  -ms-flex-align: flex-end;
  align-items: flex-end;
  background-color: #25262a;
  cursor: pointer;
  -webkit-transition: boxShadow .25s ease;
  transition: boxShadow .25s ease;
}

.c7 p {
  border: 1px solid;
  border-color: #dfe2e6;
  border-radius: 6px;
  color: #6b7480;
  height: 46px;
  padding: 0px 24px;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  font-family: Nunito Sans,sans-serif;
  text-align: center;
  margin-bottom: 44px;
}

.c7:hover,
.c7:focus {
  outline: none;
}

.c7:hover p,
.c7:focus p {
  box-shadow: 0 0 2px 3px #6b748090;
}

@media screen and (min-width:0rem) {
  .c1 {
    padding: 0px 15px;
    padding-top: 15px;
  }
}

@media screen and (min-width:37.5rem) {
  .c1 {
    padding: 0px 15px;
    padding-top: 15px;
  }
}

@media screen and (min-width:62rem) {
  .c1 {
    padding: 0px 48px;
    padding-top: 24px;
  }
}

@media screen and (min-width:0rem) {
  .c2 {
    padding: 15px;
  }
}

@media screen and (min-width:37.5rem) {
  .c2 {
    padding: 15px;
  }
}

@media screen and (min-width:62rem) {
  .c2 {
    padding: 12px 48px;
  }
}

@media (max-width:1300px) {
  .c2 {
    -webkit-flex-direction: column;
    -ms-flex-direction: column;
    flex-direction: column;
  }
}

@media (max-width:1300px) {
  .c3 {
    margin-bottom: 20px;
  }
}

@media screen and (min-width:0rem) {
  .c4 {
    padding: 15px;
  }
}

@media screen and (min-width:37.5rem) {
  .c4 {
    padding: 15px;
  }
}

@media screen and (min-width:62rem) {
  .c4 {
    padding: 40px;
  }
}

@media screen and (min-width:0rem) {
  .c4 {
    height: 100%;
  }
}

@media screen and (min-width:37.5rem) {
  .c4 {
    height: 100%;
  }
}

@media screen and (min-width:62rem) {
  .c4 {
    height: 320px;
  }
}

@media screen and (min-width:0rem) {
  .c9 {
    padding-left: 15px;
  }
}

@media screen and (min-width:37.5rem) {
  .c9 {
    padding-left: 15px;
  }
}

@media screen and (min-width:62rem) {
  .c9 {
    padding-left: 48px;
  }
}

@media (max-width:1300px) {
  .c9 {
    padding-left: 0px;
  }
}

<div
    class="c0"
    width="100%"
  >
    <h1
      class="c1"
      color="text.contrast"
      font-size="20"
      font-weight="bold"
    >
      Nova despesa
    </h1>
    <form
      action="#"
      style="width: 100%;"
    >
      <div
        class="c2"
        height="100%"
        width="100%"
      >
        <div
          class="c3"
          width="100%"
        >
          <div
            class="c4"
            height="[object Object]"
            width="100%"
          >
            <div
              class="c5"
              width="100%"
            >
              <label
                class="c6"
                color="text.contrast"
                font-weight="bold"
              >
                Recibo, cupom ou nota fiscal*
              </label>
              <div
                class="c7"
                tabindex="0"
              >
                <input
                  accept="image/jpeg, image/png"
                  autocomplete="off"
                  name="resourceUrl"
                  style="display: none;"
                  tabindex="-1"
                  type="file"
                />
                <p>
                  Selecione um arquivo do seu computador
                </p>
              </div>
              <p
                class="c8"
                color="#848a91"
              >
                A imagem deve estar no formato JPG ou PNG.
              </p>
            </div>
          </div>
        </div>
        <div
          class="c9"
          width="100%"
        >
          <div
            class="c10"
            width="100%"
          >
            <div
              class="c5"
              width="100%"
            >
              <label
                class="c6"
                color="text.contrast"
                font-weight="bold"
              >
                Tipo*
              </label>
              <div
                class="c11 css-2b097c-container"
                id="expenseTypeCode"
              >
                <div
                  class="select__control css-yk16xz-control"
                >
                  <div
                    class="select__value-container select__value-container--has-value css-g1d714-ValueContainer"
                  >
                    <div
                      class="select__single-value css-1uccc91-singleValue"
                      style="opacity: 1; transition: opacity 1ms;"
                    >
                      Selecione
                    </div>
                    <div
                      class="css-b8ldur-Input"
                    >
                      <div
                        class="select__input"
                        style="display: inline-block;"
                      >
                        <input
                          aria-autocomplete="list"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          id="react-select-2-input"
                          spellcheck="false"
                          style="box-sizing: content-box; width: 2px; border: 0px; opacity: 1; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                        <div
                          style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="select__indicators css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      class="select__indicator-separator css-1okebmr-indicatorSeparator"
                    />
                    <div
                      aria-hidden="true"
                      class="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                    >
                      <svg
                        aria-hidden="true"
                        class="css-6q0nyr-Svg"
                        focusable="false"
                        height="20"
                        viewBox="0 0 20 20"
                        width="20"
                      >
                        <path
                          d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
            <div
              class="c5"
              width="100%"
            >
              <label
                class="c6"
                color="text.contrast"
                font-weight="bold"
              >
                Moeda*
              </label>
              <div
                class="c11 css-2b097c-container"
                id="currencyCode"
              >
                <div
                  class="select__control css-yk16xz-control"
                >
                  <div
                    class="select__value-container select__value-container--has-value css-g1d714-ValueContainer"
                  >
                    <div
                      class="select__single-value css-1uccc91-singleValue"
                      style="opacity: 1; transition: opacity 1ms;"
                    >
                      Selecione
                    </div>
                    <div
                      class="css-b8ldur-Input"
                    >
                      <div
                        class="select__input"
                        style="display: inline-block;"
                      >
                        <input
                          aria-autocomplete="list"
                          autocapitalize="none"
                          autocomplete="off"
                          autocorrect="off"
                          id="react-select-3-input"
                          spellcheck="false"
                          style="box-sizing: content-box; width: 2px; border: 0px; opacity: 1; outline: 0; padding: 0px;"
                          tabindex="0"
                          type="text"
                          value=""
                        />
                        <div
                          style="position: absolute; top: 0px; left: 0px; visibility: hidden; height: 0px; overflow: scroll; white-space: pre; font-family: -webkit-small-control; letter-spacing: normal; text-transform: none;"
                        />
                      </div>
                    </div>
                  </div>
                  <div
                    class="select__indicators css-1hb7zxy-IndicatorsContainer"
                  >
                    <span
                      class="select__indicator-separator css-1okebmr-indicatorSeparator"
                    />
                    <div
                      aria-hidden="true"
                      class="select__indicator select__dropdown-indicator css-tlfecz-indicatorContainer"
                    >
                      <svg
                        aria-hidden="true"
                        class="css-6q0nyr-Svg"
                        focusable="false"
                        height="20"
                        viewBox="0 0 20 20"
                        width="20"
                      >
                        <path
                          d="M4.516 7.548c0.436-0.446 1.043-0.481 1.576 0l3.908 3.747 3.908-3.747c0.533-0.481 1.141-0.446 1.574 0 0.436 0.445 0.408 1.197 0 1.615-0.406 0.418-4.695 4.502-4.695 4.502-0.217 0.223-0.502 0.335-0.787 0.335s-0.57-0.112-0.789-0.335c0 0-4.287-4.084-4.695-4.502s-0.436-1.17 0-1.615z"
                        />
                      </svg>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>
          <div
            class="c12"
          >
            <label
              for="notes"
            >
              Título da despesa*
            </label>
            <input
              id="notes"
              name="notes"
              placeholder="Digite..."
              type="text"
              value=""
            />
          </div>
          <div
            class="c5"
            width="100%"
          >
            <label
              class="c6"
              color="text.contrast"
              font-weight="bold"
            >
              Data do comprovante*
            </label>
            <div
              class="c13"
            >
              <div
                class="react-datepicker-wrapper"
              >
                <div
                  class="react-datepicker__input-container"
                >
                  <input
                    class=""
                    id="cardDate"
                    placeholder="DD/MM/YYYY"
                    type="text"
                    value=""
                  />
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="c14"
        height="77px"
        width="100%"
      >
        <button
          class="c15"
          color="#FFFFFF"
          type="button"
        >
          Cancelar
        </button>
        <button
          class="c16"
          color="#FFFFFF"
          type="submit"
        >
          Salvar
        </button>
      </div>
    </form>
  </div>
</DocumentFragment>
`;
